summit-demo/README.md

```diff
@@ -0,0 +1,120 @@
+# This is how I totally not faked the demo
+
+## Create project
+
+1. Go here https://git.stg.centos.org/
+
+2. Log in. (use prod credentials, there doesn't seem to be stg.accounts)
+
+3. Create an API token.
+
+Now the code I ran:
+
+```python
+In [1]: import ogr
+
+In [2]: s = ogr.PagureService(token="da_token", instance_url="https://git.stg.centos.org/")
+
+In [4]: s.get_api_url()
+Out[4]: 'https://git.stg.centos.org//api/0/'
+
+In [9]: p = s.project_create(repo="rpm", namespace="source-git")
+```
+
+And voilà... https://git.stg.centos.org/source-git/rpm
```

Comment:
Probably nitpick: I had the impression, that the namespace is going to be called `src` (instead of `source-git`), though my memory might cheat me.

Reply:
I would prefer src as well. We can talk to Brian about this.